### DISCUSSION TITLE
Jagged edges on objects of a video

### DISCUSSION THREAD

**Discussion by P4ST4N4G4** [ntsc-rs/ntsc-rs#194] [General]
So when I use the "scale to x scanlines" options most videos get "jagged" edges on objects (I'm not referring to dot crawl) and I'm not sure if that's intended behaviour since if I downscale the video on an external program to 640x480 and then use that as the base the resulting picture doesn't have the "jagged" edges on things. 
Is it something intentional?

This is the video using the internal "scale to x scanlines" option:
![nativedownscaling](https://github.com/user-attachments/assets/98d86071-806d-4de1-ae6c-ef4da6f8fce1)

This is the same video (same preset too) but first downscaled with an external program: 
![externaldownscale](https://github.com/user-attachments/assets/dca376ad-1254-4742-ae17-017a24478bcd)

The most noticable part is the tower on the center of the frame and the tracks on front of the camera.

**Comment by Degamisu** [comment]
My guess is that NTSC-RS doesnt really downscale, or uses `Nearest Neighbor` filtering. That would cause much more interference than the external software downscaling with `Bilinear` filtering!

**Comment by valadaptive** [comment]
You can now change the scaling filter in v0.9.0.
